src/applications/vre/28-1900/containers/OrientationWizardContainer.jsx

```diff
@@ -54,12 +54,13 @@ const OrientationWizardContainer = props => {
           apply. If you are, we encourage you to complete the orientation before
           you apply.
         </p>
-        <p>
+        <p id="skip-wizard-description">
           <strong>If you already know you want to apply for VR&E</strong>, you
           can go directly to the online application without answering the
           questions below.{' '}
           <a
             href={CHAPTER_31_ROOT_URL}
+            aria-describedby="skip-wizard-description"
```

Comment:
I was a bit concerned on how this would perform if the with the `aria-descirbedby` nested within the id it's referencing, but [it seems to be working fine on this codepen I whipped up](https://codepen.io/joshkimux/pen/rNjMzxo). Let's keep an eye on this for when it goes into staging.